docker-compose.yml

```diff
@@ -1,14 +1,20 @@
 version: "3.4"
 
 services:
-  # for development purposes
   ci: &ci
     build: .
     command: ['composer', 'ci']
 
-  tests-local:
+  tests-local: &tests-local
```

Comment:
Vim ze to tu uz bylo, ale ty services by se mely jmenovat `dev`, ne `test-local` (to je nejaky legacy bordel)

Reply:
Jo, taky se mi to nepozdávalo, přejmenuju.